### PR TITLE
Disable ecs

### PR DESCRIPTION
**Pull request by max-jacobs**
To be merged in to master once `iam-roles` has been merged.

This PR makes it possible to switch off ECS for Beekeeper.

### CODE REVIEW

File: main/variables.tf [hunk 1]
```diff
@@ -12,6 +12,12 @@ variable "instance_name" {
   default     = ""
 }
 
+variable "instance_type" {
+  description = "Service to run Beekeeper on. Supported services: `ecs` (default). Leaving this blank will deploy auxilliary components (e.g. RDS, SQS etc.) and will output IAM policies which can used to create the required roles."
```
Comment (abhimanyugupta07) [inline main/variables.tf]:
```suggestion
  description = "Service to run Beekeeper on. Supported services: `ecs` (default). Leaving this blank will deploy auxilliary components (e.g. RDS, SQS etc.) and will output IAM policies which can used to create the required roles for EKS."
```

may be better ?

File: main/README.md [hunk 2]
```diff
@@ -4,49 +4,60 @@ Terraform deployment scripts for [Beekeeper](https://github.com/ExpediaGroup/bee
...
+| receive\_wait\_time\_seconds | SQS receive wait time (s). | string | `"20"` | no |
+| scheduler\_delay\_ms | Delay between each cleanup job that is scheduled in milliseconds. | string | `"300000"` | no |
+| subnets | Subnets in which to install Beekeeper. | list | n/a | yes |
+| vpc\_id | VPC in which to install Beekeeper. | string | n/a | yes |
+
+## Outputs
+
+| Name | Description |
+|------|-------------|
+| beekeeper\_cleanup\_config | Rendered Spring config for Cleanup application. |
```
Comment (abhimanyugupta07) [inline main/README.md]: one output is missing here:

```output "sg" {
  value       = "${aws_security_group.beekeeper_sg.id}"
  description = "ID of Beekeeper SG."
}```

Comment (max-jacobs) [reply main/README.md]: thanks, updated

File: main/variables.tf (hunk 1 above)
Comment (max-jacobs) [reply main/variables.tf]: Fixed

File: main/README.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ If the chosen `db_password_strategy` is `aws-secrets-manager`, this terraform mo
 
 | Name | Description | Type | Default | Required |
 |------|-------------|:----:|:-----:|:-----:|
-| allowed\_s3\_buckets | List of S3 Buckets to which Beekeeper will have read-write access. | list | `<list>` | no |
+| allowed\_s3\_buckets | List of S3 Buckets to which Beekeeper will have read-write access. | list | n/a | yes |
```
Comment (abhimanyugupta07) [inline main/README.md]: This needs to be not a required field.

Comment (max-jacobs) [reply main/README.md]: Done

**Pull request merged**